tools.yml

```diff
@@ -2558,3 +2561,21 @@ tools:
     mem: 32
   wig_to_bigWig:
     mem: 10
+destinations:
+  default:
+    params:
+      tpv_cores: '{cores}'
+      tpv_gpus: '{gpus}'
+      tpv_mem: '{mem}'
+    abstract: true
+  tpvdb_slurm:
+    context:
+      time: ''
```

Comment:
I'm wondering if these variable names are too generic. 'time' in particular could collide with python's time library

Reply:
That's a good point. I've renamed time for now, but perhaps we could just prefix everything with `slurm_` instead?

Reply:
I like the prefixing idea.